tree_queries/compiler.py

```diff
@@ -139,6 +139,12 @@ def as_sql(self, *args, **kwargs):
         }
 
         if "__tree" not in self.query.extra_tables:  # pragma: no branch - unlikely
+            tree_params = params.copy()
+            # use aliased table name "UX"
+            base_table = self.query.__dict__.get("base_table", None)
```

Comment:
```suggestion
            base_table = self.query.__dict__.get("base_table")
```

The default value of `dict.get` is always `None`, you don't have to provide this.